completions/zsh/_aur

```diff
@@ -48,7 +48,6 @@ _aur_build_sync_args=(
     '(-n --no-confirm)'{-n,--no-confirm}'[do not wait for user input when installing or removing build dependencies]'
     '(-r --rmdeps)'{-r,--rmdeps}'[remove dependencies installed by makepkg]'
     '(-S --sign --gpg-sign)'{-S,--sign,--gpg-sign}'[sign build packages and the database with gpg]'
-    '(-T --temp)'{-T,--temp}'[build in a temporary container]'
```

Comment:
I'd leave in the completions until `chroot--standalone` becomes the default implementation